Polyhedron/demo/Polyhedron/Scene_polyhedron_item.cpp

```diff
@@ -1835,3 +1843,15 @@ void Scene_polyhedron_item::set_flat_disabled(bool b)
   invalidateOpenGLBuffers();
   itemChanged();
 }
+
+void Scene_polyhedron_item::itemAboutToBeDestroyed(Scene_item *item)
+{
+  Scene_item::itemAboutToBeDestroyed(item);
+  if(d && item == this)
+  {
+    delete d->poly;
+    delete d->targeted_id;
+    delete d;
+    d=NULL;
```

Comment:
Those lines are strange... the destructor of `d` does not destroy `poly` and `targeted_id`?